### DISCUSSION TITLE
Increase frequency of link checking service or make link checking on-demand

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#104] [General Suggestions]
### What is the problem that your suggestion solves?

The link checking service doesn't run frequently and doesn't re-check links that have already been checked. Consequently, the data returned by the link checking service can be outdated. 

### What solution might meet your needs?

- Run link checking more frequently and run it on DOIs that have already been checked. 
- An on-demand link checking service. This will greatly help organizations in making sure their DOIs resolve correctly, especially for repositories minting many DOIs.

### Is there anything else you would like to share?

[Link for DataCite staff](https://docs.google.com/spreadsheets/d/16jQnuW_DMgbb-74Y5TByUG4IkHdvJSZ0Fe_qxQ4Xl6w/edit?gid=65276345#gid=65276345&range=381:381)

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by adambuttrick** [comment]
Since this came up relative to some CDL discussions in the America Experts Group, what we're looking at doing is making this user configurable in our service, [EZID](https://ezid.cdlib.org), with some sane defaults. If users know links are broken, but need time to repair, or want to do so iteratively relative to a change, its good to be able to have options about when and to what extent this should occur. Otherwise, you can have cases where you're repeatedly hammering something knowing it won't work, but with no way to intervene.

**Comment by NelliTaller** [comment]
We would strongly welcome it if links could be checked on-demand. 
Unfortunately, no links are currently checked and the old data is not usable. Greetings from Germany, Nelli

**Reply by wlongabaugh** [reply]
I would also appreciate being able to run the scan on demand. We have very old and outdated scan info for a number of our records.
